### ISSUE TITLE
Lock Semesters on Mark Ui

### ISSUE THREAD

**Issue by M4R1KU** [outcobra/outstanding-cobra#265]
I've had the idea that we could make a semester lockable on the Marks Ui.

This would mean that it can't be modified in any way when it is locked.
Currently I just lock semesters that are not active but this takes the control from the user.

I'd like to hear your thoughts @jmesserli @needToRoll

**Comment by jmesserli** [comment]
I assume this is to reduce the amount of displayed semesters? In that case
I agree that this is a good idea.

On Jul 29, 2017 17:11, "Mario Kunz" <notifications@github.com> wrote:

I've had the idea that we could make a semester lockable on the Marks Ui.

This would mean that it can't be modified in any way when it is locked.
Currently I just lock semesters that are not active but this takes the
control from the user.

I'd like to hear your thoughts @jmesserli <https://github.com/jmesserli>
@needToRoll <https://github.com/needtoroll>

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub
<https://github.com/outcobra/outstanding-cobra/issues/265>, or mute the
thread

**Comment by M4R1KU** [comment]
This was not the original intent of it but we could surely combine those
two things.

Or we could make also a sort of archive of semetser marks.
And so it could be more convenient to manage it when you use the tool over
multiple years.

Joel Messerli <notifications@github.com> schrieb am Sa., 29. Juli 2017,
16:24:

> I assume this is to reduce the amount of displayed semesters? In that case
> I agree that this is a good idea.
>

**Comment by jmesserli** [comment]
What was your original intention?

**Comment by M4R1KU** [comment]
@needToRoll said that non-active semesters should not be editable and I thought that this is not really the best thing to do and so I thought about locking semesters but with the possibility to unlock them again to make changes even if the semester is not active

**Comment by jmesserli** [comment]
Imho that doesn't really add much value, but if you both think it does, I
won't be standing in the way :)

**Comment by M4R1KU** [comment]
You are right there is very very very little value in this feature. We should probably discuss later around a table or so.

**Comment by jmesserli** [comment]
You are probably right 👍
